packages/cli/tests/commonlib/cliHelper.ts

```diff
@@ -113,4 +113,29 @@ export class CliHelper {
       console.log(`[Successfully] ${message}`);
     }
   }
+
+  static async getUserSettings(key: string, projectPath: string, env: string): Promise<string> {
+    let value = "";
+    const result = await execAsync(`teamsfx config get ${key} --env ${env}`, {
+      cwd: projectPath,
+      env: process.env,
+      timeout: 0,
+    });
+
+    const message = `get user settings in ${projectPath}. Key: ${key}`;
+    if (result.stderr) {
+      console.error(`[Failed] ${message}. Error message: ${result.stderr}`);
+    } else {
+      const arr = (result.stdout as string).split(":");
+      if (!arr || arr.length <= 1) {
+        console.error(
+          `[Failed] ${message}. Failed to get value from cli result. result: ${result.stdout}`
+        );
+      } else {
+        value = arr[1].trim() as string;
+        console.log(`[Successfully] ${message} Value: ${value}`);
```

Comment:
though it's for test purpose, not sure whether we should log the values, which might be secrets